### PR TITLE
feat: seo api

### PR DESCRIPTION
**Pull request by AlanSyue**
# Description
Create an API for SEO use.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] see the API in stage (https://api-stage.itseed.tw/seos)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/components/meta/seo.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      "maxLength": 60
+    },
+    "description": {
+      "type": "string",
+      "required": true,
+      "maxLength": 160
+    },
+    "keywords": {
+      "type": "string"
+    }
```
Comment (Nick0603) [inline app/components/meta/seo.json:22]: We should contain more meta tag in it. Like as `og:title`, `og:description` adn `og:image`

**Review by Nick0603 — CHANGES_REQUESTED** [review]

**Comment by AlanSyue** [comment]
Add following meta tag
- og:title
- og:description
- og:url
- og:image

Remove `keywords` tag

File: app/package.json [hunk 2]
```diff
@@ -21,7 +21,6 @@
     "strapi-plugin-documentation": "^3.6.8",
     "strapi-plugin-email": "3.6.8",
     "strapi-plugin-i18n": "3.6.8",
-    "strapi-plugin-sentry": "^3.6.8",
```
Comment (Nick0603) [inline app/package.json:24]: Revert the adjustion

**Review by Nick0603 — APPROVED** [review]
LGTM

**Pull request merged**